django_rq/.kdev4/django_rq.kdev4

```diff
@@ -0,0 +1,17 @@
+[Buildset]
```

Comment:
This file shouldn't be committed. Could you please remove this file from the repo?